soc/posix/inf_clock/Kconfig

```diff
@@ -28,3 +28,13 @@ config NATIVE_SIMULATOR_PRIMARY_MCU_N
 	  This MCU will for example have its tests command line parameters presented
 	  without any prefix. Note that an MCU being primary does not imply it will be
 	  the first one to boot.
+
+config NATIVE_SIMULATOR_EXTRA_IMAGE_PATHS
```

Comment:
for me a path sounds like something that should rather be in cmake, do you agree?

Reply:
discussed offline that this makes it easy for users to find and more evident that it is an option they can set it, and it is used similarly in quite a few other places.